README.md

```diff
@@ -0,0 +1,120 @@
+# Phat QuickJS
+
+A phat contract that ports the QuickJS engine to pink environment.
+
+## Build (Ubuntu 20.04)
+
+### Prerequirements
+
+- See https://github.com/Phala-Network/phat-contract-examples for the phat contract development environment preparation.
+- Some C toolchain also needed if it wasn't installed
+
+  ```bash
+  apt install clang clang-dev make
+  ```
+
+### Build commands
+
+```bash
+git clone https://github.com/Phala-Network/phat-quickjs.git
+cd phat-quickjs
+make
+```
+
+If no error happens, it should output normal compiled ink contract files in the path `target/ink`:
+
+```
+$ ls target/ink/
+CACHEDIR.TAG  metadata.json  qjs.contract  qjs.wasm  release  wasm32-unknown-unknown
+```
+
+## Usage
```

Comment:
This is the low level usage, right? We can wrap it as a driver and expose a simple function call.